lib/Conversion/SeqToSV/SeqToSV.cpp

```diff
@@ -404,6 +405,32 @@ void SeqToSVPass::runOnOperation() {
 
   FirMemLowering memLowering(circuit);
 
+  auto randomInitFragmentName =
+      FlatSymbolRefAttr::get(context, "RANDOM_INIT_FRAGMENT");
+  auto randomInitRegFragmentName =
+      FlatSymbolRefAttr::get(context, "RANDOM_INIT_REG_FRAGMENT");
+  auto randomInitMemFragmentName =
+      FlatSymbolRefAttr::get(context, "RANDOM_INIT_MEM_FRAGMENT");
+
+  auto regFragments = ArrayAttr::get(
+      context, {randomInitFragmentName, randomInitRegFragmentName});
+  auto memFragments = ArrayAttr::get(
+      context, {randomInitFragmentName, randomInitMemFragmentName});
+
+  auto addFragments = [&](HWModuleOp module, ArrayAttr fragments) {
+    constexpr StringRef attrName = "emit.fragments";
```

Comment:
Nit: How about adding it to the headers and using something like, `getFragmentsAttrName()` , instead of hardcoding the attrname in different files.